pkg/js/js.go

```diff
@@ -87,10 +88,11 @@ func require(call otto.FunctionCall) otto.Value {
 	// Record the old currentDirectory so that we can return there.
 	currentDirectoryOld := currentDirectory
 	// Record the directory path leading up to the file we're about to require.
-	currentDirectory = filepath.Clean(filepath.Dir(cleanFile))
+	currentDirectory = filepath.Dir(cleanFile)
 
 	printer.Debugf("requiring: %s (%s)\n", file, relFile)
-	data, err := ioutil.ReadFile(relFile)
+	// quick fix, by replacing to linux slashes, to make it work with windows paths too.
+	data, err := ioutil.ReadFile(filepath.ToSlash(relFile))
```

Comment:
Suggestion: Require the user to always specify a Unix-style file path, but use FromSlash() so that on Windows systems it works as expected.

Reply:
`ioutil.ReadFile()` is also able to use Windows file paths without issues. So this also works on Windows machines. (tested myself)

Reply:
I haven't put much thought into it.  My point is that the ideal situation would be one file that works (unchanged) on windows, linux, and macos.  Would both require_glob("foo/bar") and require_blog(`foo\bar`) both work in all situations?  If not, I'd rather document that people should specify things the Linux way, and have the code use FromSlash.

Reply:
Yeah, I get the idea. I might not have specified this clear enough: `ToSlash` forces to use Unix paths (`/`), no matter which OS. And as `ReadFile` can work with unix-paths on Windows as well, it works cross-platform on Linux and Windows.

Reply:
Ah, I see!  Thanks for the clarification!